tile/targets/BUILD

```diff
@@ -11,7 +11,6 @@ gencfg(
         "//tile/targets/gpu:nvidia.json",
         "//tile/targets/gpu:intel_gen9.json",
     ] + select({
```

Comment:
The way to do this would be to remove the `select()` entirely and merge the default `"//tile/targets/cpu:cpu.json"` into the `srcs` list.